core/string/ustring.h

```diff
@@ -652,6 +652,12 @@ class String {
 	}
 };
 
+// This exposes `String` as `Godot::String`.
+// This solves godotengine/godot#104038
```

Comment:
I believe the convention for issues is GH-xxx:

```suggestion
// This solves GH-104038
```